README.md

```diff
@@ -289,7 +290,8 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://www.ls.graphics/illustrations/clak"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808599-e430ada2-18af-4581-bfe7-c70b2b832853.png' alt="Clak_freemium_illustrations" /></a> | [Clak](https://www.ls.graphics/illustrations/clak)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br>• AI<br><br> ✌️Author <br> Ls Graphics                                               |
 | <a href="https://characterz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/194710390-b436f918-f494-41a0-bda1-76f205e95937.png' alt="make_this_appropriate" /></a> | [CHARACTERZ 2](https://characterz.design/)  <br><br> ✨ Freemium  <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PSD<br>• AI<br>• PNG<br>• Blender 3D<br><br> ✌️Author <br> ThreeDee                                                            |
 | <a href="https://emojiz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/195575740-741a625a-7631-4e4e-9d24-5db0f1f30932.png' alt="Emojiz_freemium_illustrations" /></a> | [Emojiz](https://emojiz.design/)  <br><br> ✨ Freemium  <br><br>Format <br> • Sketch<br>• Adobe Photoshop<br>• Adobe Illustrator<br>• Adobe XD<br>• Figma<br>• InVision Studio<br>• Blender 3D<br><br> ✌️Author <br> ThreeDee                       |
-| <a href="https://www.handz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808197-3d32a0ff-c114-46d5-9e17-2e3a688096bc.png' alt="Handz_3D_Freemium_Illustrations" /></a> | [Handz 3D](https://www.handz.design/)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PSD<br>• AI<br>• PNG<br><br> ✌️Author <br> Handz Design                                                                        |
+| <a href="https://www.handz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808197-3d32a0ff-c114-46d5-9e17-2e3a688096bc.png' alt="Handz_3D_Freemium_Illustrations" /></a> | [Handz 3D](https://www.handz.design/)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PSD<br>• AI<br>• PNG<br><br> ✌️Author <br> Handz Design
+| <a href="https://lordicon.com/icons/wired/lineal"><img width='700' src='https://user-images.githubusercontent.com/97742841/196000594-b1056def-415f-4fb9-bfbb-f745234a262d.png' alt="Wired Lineal" /></a> | [Lordicon](https://lordicon.com/icons/wired/lineal)  <br><br> ✨ Freemium  <br><br>Format <br> • PNG<br>• SVG<br>• GIF<br>•JSON<br>•Embed HTML<br>•APNG<br>•MP4<br>•WebP<br><br> ✌️Author <br> Lordicon |
```

Comment:
- add format `EPS` & `After Effects
- change alt text to `Wired_Lineal_freemium_illustrations`
- change the title to Wired Lordicon Icons

Reply:
@intozen 
You haven't completed the fourth point i.e. arrange in alphabetical order